src/content/get-dart/index.md

```diff
@@ -1,131 +1,142 @@
+| Windows  | {{yes}} | {{yes}} | {{no}}  | {{beta}}  | {{no}}   | [10] (32-bit, 64-bit), [11][]     |
+| Linux    | {{yes}} | {{yes}} | {{yes}} | {{yes}}   | {{beta}} | [Debian stable][], [Ubuntu LTS][] |
+| macOS    | {{no}}  | {{yes}} | {{na}}  | {{yes}}   | {{na}}   | {{macversions}}                   |
+
+{:.table .table-striped}
+
+{{yes}} Supported in all channels.  
+{{no}} Unsupported in all channels.  
+{{beta}} Supported in Dev, Beta channels only.  
+{{na}} No version exists.
+
+## Choose an install option
+
+To install and update the Dart SDK from the stable channel,
+choose one of the following options:
+
+1. [Use a package manager](#install) (Recommended).
+1. Use a [Dart Docker image][dart-docker]
+1. Download from the [SDK Archive](/get-dart/archive)
+1. [Build the SDK from source][build-source]
+
 {% comment %}
 NOTE to editors: Keep the zip file link as the last thing in the paragraph,
 so it's easy to find (but not more tempting than package managers).
 {% endcomment %}
 
-*Note*: The Flutter SDK includes the full Dart SDK,
-and has Dart's [`dart`](/tools/dart-tool) command-line interface
-in its `bin` folder.
-
 :::warning Notice
 {% include './archive/_sdk-terms.md' %}
 :::
 
+If you've installed or plan to [install the Flutter SDK][install-flutter],
+it includes the full Dart SDK. The Flutter SDK includes the
+[`dart`](/tools/dart-tool) CLI tool in Flutter's `bin` folder.
+
+## Install the Dart SDK using a package manager {:#install}
+
+Install the Dart SDK using the package manager for your platform.
+
 <ul class="tabs__top-bar">
   <li class="tab-link current" data-tab="tab-sdk-install-windows">Windows</li>
   <li class="tab-link" data-tab="tab-sdk-install-linux">Linux</li>
   <li class="tab-link" data-tab="tab-sdk-install-mac">macOS</li>
 </ul>
-<div id="tab-sdk-install-windows" class="tabs__content current">
-
+<div id="tab-sdk-install-windows" class="tabs__content current" markdown="1">
 {% include './_windows.md' %}
-
 </div>
-<div id="tab-sdk-install-linux" class="tabs__content">
-
+<div id="tab-sdk-install-linux" class="tabs__content" markdown="1">
 {% include './_linux.md' %}
-
 </div>
-<div id="tab-sdk-install-mac" class="tabs__content">
-
+<div id="tab-sdk-install-mac" class="tabs__content" markdown="1">
 {% include './_mac.md' %}
-
 </div>
 
-## System requirements
-
-The Dart SDK is supported on Windows, Linux, and macOS.
-
-### Windows
-
-* **Supported versions:** Windows 10 and 11.
-* **Supported architectures:** x64, IA32, ARM64.<br>
-  Support for ARM64 is in preview, and is available only in the dev and beta channels.
-
-### Linux
-
-* **Supported versions:** [Debian stable][] and [Ubuntu LTS][] under standard support.
-* **Supported architectures:** x64, IA32, ARM64, ARM, RISC-V (RV64GC).<br>
-  Support for RISC-V is in preview, and is available only in the dev and beta channels.
-
-### macOS
-
-* **Supported versions:** Latest three major versions.
```

Comment:
I'm pulling from a JSON file that this page then inserts the latest three versions. The rule is still there, but written explicitly.

Reply:
The issue is that we're no longer tell our customers that that is our policy. So they could assume that our policy is just to add and a year from now it will list four versions instead of three.